.vsts-ci/spelling.yml

```diff
@@ -35,6 +35,15 @@ phases:
       sudo npm install -g markdown-spellcheck@0.11.0
     displayName: Install mdspell
     condition: succeededOrFailed()
+    
+  - bash: |
+      sudo npm install -g markdown-link-check@3.7.2
```

Comment:
Warning: this version must stay static on `3.7.2`!

Reply:
Is it just a hashtag?

Reply:
yes